students/BrianNisonger/session02/fizzbuzz.py

```diff
@@ -0,0 +1,17 @@
+def FizzBuzz():
+	i=1
```

Comment:
Looks like you're using tabs instead of spaces?  See if you can get your editor to default to 4 spaces when hitting tab for all indentions in Python files.